source/displayModel.py

```diff
@@ -200,7 +200,15 @@ def getWindowTextInRect(bindingHandle, windowHandle, left, top, right, bottom,mi
 	characterLocations = []
 	cpBufIt = iter(cpBuf)
 	for cp in cpBufIt:
-		characterLocations.append(RectLTRB(wcharToInt(cp), wcharToInt(next(cpBufIt)), wcharToInt(next(cpBufIt)), wcharToInt(next(cpBufIt))))
+		left, top, right, bottom = (
+			wcharToInt(cp),
+			wcharToInt(next(cpBufIt)),
+			wcharToInt(next(cpBufIt)),
+			wcharToInt(next(cpBufIt))
+		)
+		if right < left:

```

Comment:
Do you know under what circumstances this becomes reversed? can you put a comment here explaining this at least?

Reply:
I'm not 100% sure, but my theory is that this happens in case of left to right languages.

Reply:
Thinking more about this, I think it would be better to split this specific fix out into a new pr.